### PR TITLE
Colu-961 decrease logs

### PR DESCRIPTION
**Pull request by oleiba**

### CODE REVIEW

File: bin/casimir.js [hunk 1]
```diff
@@ -8,10 +8,11 @@ properties.server.favicon = __dirname + '/../' + properties.server.favicon
 properties.server.compression = process.env.COMPRESSION || properties.server.compression
 properties.engine.view_folder = __dirname + '/../' + properties.engine.view_folder
 properties.engine.static_folder = __dirname + '/../' + properties.engine.static_folder
+properties.debug = process.env.DEBUG === 'true'
```
Comment (tal-beja) [inline bin/casimir.js]: I can't use the properties file to set debug mode to true?

File: package.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
     "bitcoin-async": "^1.0.0",
     "bitcoinjs-lib": "^1.5.6",
     "bs58": "^3.0.0",
-    "casimircore": "^0.9.0",
+    "casimircore": "^0.10.0",
```
Comment (tal-beja) [inline package.json:11]: the updated casimir wont break the engine?

**Review by tal-beja — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (oleiba) [reply package.json:11]: no, engine currently has `~0.9.2`, so it receives the last 0.9.X.

File: bin/casimir.js (hunk 1 above)
Comment (oleiba) [reply bin/casimir.js]: no

Comment (oleiba) [reply bin/casimir.js]: now possible from [this commit](https://github.com/Colored-Coins/Colored-Coins-Block-Explorer/pull/8/commits/3cc6bb5fb40a9bc3e620b55db2df866a2af676c3)

**Review by tal-beja — APPROVED** [review]

**Review by tal-beja — APPROVED** [review]

**Pull request merged**